ports/protobuf/portfile.cmake

```
@@ -27,24 +27,33 @@ endif()
 
 if (VCPKG_DOWNLOAD_MODE)
     # download PKGCONFIG in download mode which is used in `vcpkg_fixup_pkgconfig()` at the end of this script.
-    # download it here because `vcpkg_configure_cmake()` halts execution in download mode when running configure process.
+    # download it here because `vcpkg_cmake_configure()` halts execution in download mode when running configure process.
     vcpkg_find_acquire_program(PKGCONFIG)
 endif()
 
```

```suggestion

set(protobuf_VISIBILITY_FLAGS )
```
We should declare `protobuf_VISIBILITY_FLAGS` first in oder to prevent it's not defined.